src/server/.tests/routes/utilities/responseManager.spec.js

```diff
@@ -0,0 +1,43 @@
+const responseManager = require('../../../routes/controllers/utilities/responseManager')
+const assert = require('chai').assert
+const httpMocks = require('node-mocks-http')
+const EventEmitter = require('events').EventEmitter
+
+let suite ={}
+
+describe('responseManager', () => {
+	beforeEach(() => {
+		suite = {}
+		suite.responeMock = httpMocks.createResponse({
+			eventEmitter: EventEmitter
+		})
+	})
+
+	describe('#createResponse', () => {
```

Comment:
Shouldn't we call it `sendResponse`? Tests show that it sends a response, while create might look like you just create an object.

Reply:
ok